base/src/main/java/com/austindroids/austinfeedsme/data/FilterableEventDataSource.java

```diff
@@ -0,0 +1,6 @@
+package com.austindroids.austinfeedsme.data;
+
+public interface FilterableEventDataSource extends EventsDataSource {
+    void getEvents(LoadEventsCallback callback, boolean futureEvents,
```

Comment:
Consider using 3 separate functions instead of these random booleans.

Reply:
good call - this method kind of grew out of a single method